todo_app/data/mongo_items.py

```diff
@@ -0,0 +1,62 @@
+import requests, os, json, pymongo, dotenv
+from pymongo import MongoClient
+from requests import api
+from todo_app.data.todoitem import ToDoItem
+from requests.api import post
+from bson import ObjectId
+import certifi
+
+
+ca = certifi.where()
+connectionn_string = os.environ.get("PRIMARY_CONNECTION_STRING")
+client = MongoClient(connectionn_string, tlsCAFile=ca)
```

Comment:
Separately to the below, you're suffering from one of the more annoying bugs I've found through no fault of your own - mongomock mocks the pymongo library in such a way that it fails to intercept individually imported classes/functions; what I mean by that is this fails for me currently but if I switch to:
```suggestion
client = pymongo.MongoClient(connectionn_string, tlsCAFile=ca)
```
Then it helps!

Reply:
thanks jack - done :)